src/store.cc

```diff
@@ -227,6 +227,19 @@ StoreEntry::bytesWanted (Range<size_t> const aRange, bool ignoreDelayPools) cons
     return mem_obj->mostBytesWanted(aRange.end, ignoreDelayPools);
 }
 
+bool
+StoreEntry::hasParsedReplyHeader() const
+{
+    if (mem_obj) {
+        const auto &reply = mem_obj->baseReply();
+        if (reply.pstate == Http::Message::psParsed) {
+            debugs(20, 7, reply.hdr_sz);
```

Comment:
(Eventual) access to hdr_sz is one of the two primary reasons for calling this method. Printing hdr_sz value helps in triage, but I do not insist on this additional debugging. If you are against this debugging, please let me know.